pythonic-query.py

```diff
@@ -0,0 +1,172 @@
+from datetime import datetime
+import time
+import subprocess
+import json
+import requests
+
+studyid_counter = 0
+formatted_datetime = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+
```

Comment:
```suggestion
gxa_url = f"https://www.ebi.ac.uk/gxa/json/experiments/"
```

Adds a global url variable to be used throughout script